app/src/main/java/org/mozilla/fenix/settings/address/view/AddressEditorView.kt

```diff
@@ -116,9 +93,9 @@ class AddressEditorView(
             streetAddress = binding.streetAddressInput.text.toString(),
             addressLevel3 = "",
             addressLevel2 = "",
-            addressLevel1 = "",
+            addressLevel1 = binding.subregionDropDown.selectedItem.toString(),
             postalCode = binding.zipInput.text.toString(),
-            country = binding.countryDropDown.selectedItem.toString().toCountryCode(),
+            country = binding.countryDropDown.toString().toCountryCode(),
```

Comment:
was the `selectedItem` call accidentally dropped?

Reply:
Oh definitely 😬 test should've definitely caught that as well, I'll make sure that's updated